spring-security-modules/spring-security-web-boot-3/src/main/java/com/baeldung/logging/SecurityConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.logging;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Value("${spring.websecurity.debug:false}")
+    boolean webSecurityDebug;
+
+    @Override
+    public void configure(WebSecurity web) {
+        web.debug(webSecurityDebug);
+    }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests()
+                .antMatchers("/**")
+                .permitAll();
```

Comment:
Two space indent on line continuations

Reply:
thanks, @eric-martin, I was just checking but the eclipse code formatted is no longer part of the tutorials project. How can I apply code formatting with Intellij?

Reply:
https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle

Reply:
ok, could we use a check for formatting while building? It would prevent this kind of error. I've been using this https://github.com/commitd/speedy-spotless

Reply:
@lucaCambi77  - Can you send an email to team@baeldung.com with your suggestion? I'm not familiar with that.